web/virtual_lab/static/virtual_lab/js/chemistry/titration.js

```
@@ -0,0 +1,117 @@
+    dctx.beginPath(); dctx.arc(x,y,6,0,2*Math.PI); dctx.fill();
+    y += 5;
+    if (y > 110) {
+      clearInterval(id);
+      setTimeout(()=> dctx.clearRect(0,0,600,400), 50);
+    }
+  }, 20);
+}
+
+// Start titration
+function startTitration() {
+  const aC = parseFloat(document.getElementById('acid-conc').value);
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)